bundles/org.openhab.binding.bluetooth.generic/src/main/java/org/openhab/binding/bluetooth/generic/internal/GenericBluetoothHandler.java

```diff
@@ -182,6 +182,7 @@ private void updateThingChannels() {
         }
     }
 
```

Comment:
We‘ve run into that in other places. I would suggest `Objects.requireNonNull(...)` in the Return Statement like we have used in the other places.

Reply:
done